lib/gsheets/gsheets.js

```diff
@@ -23,7 +21,7 @@ var TOKEN_PATH = TOKEN_DIR + 'sheets.googleapis.com-nodejs-pwmetrics.json';
 function authenticate() {
   return new Promise((resolve, reject) => {
     // Load client secrets from a local file.
-    fs.readFile('client_secret.json', 'utf8', function processClientSecrets(err, content) {
+    fs.readFile('./lib/gsheets/client_secret.json', 'utf8', function processClientSecrets(err, content) {
```

Comment:
eh. i'm thinking of like a `client_secret_contents` property and then just paste in the content of that original file.. so you just have one file total.